docs: Update the downgrade documentation for systemd (#386)
prysmaticlabs/documentation: website/versioned_docs/version-2.0.3/prysm-usage/staying-up-to-date.md

To run our latest release with Bazel, you can look up our [releases page](https://github.com/prysmaticlabs/prysm/releases), look at the release tag you want to run, such as v1.0.5, then do `git checkout v1.0.5`. Afterwards, you can re-run your beacon chain and validator as you ran them earlier with Bazel.

**Using Systemd**
  

> Edit the systemd files for both validator (`/etc/systemd/system/beacon.service`) and beacon (`/etc/systemd/system/beacon.service`). The filename  depends on what you used when you installed, if you forgot the name, just `ls` that directory (`/etc/systemd/system/`) and edit them both. Add the `Environment` key under the `[Service]` group to have `Environment     =  USE_PRYSM_VERSION=v2.0.2`

Reviewer: ```suggestion
Edit the systemd files for both validator (`/etc/systemd/system/validator.service`) and beacon (`/etc/systemd/system/beacon.service`). The filename  depends on what you used when you installed, if you forgot the name, just `ls` that directory (`/etc/systemd/system/`) and edit them both. Add the `Environment` key under the `[Service]` group to have `Environment     =  USE_PRYSM_VERSION=v2.0.2`
```

  
Example for the beacon chain:
```
[Unit]
Description     = Ethereum Beacon Chain Service
Wants           = network-online.target
After           = network-online.target

[Service]
Type            = simple
User            = eth
ExecStart       = /home/eth/prysm/prysm.sh beacon-chain --config-file=/etc/prysm/beacon-chain.yaml
Restart         = on-failure
TimeoutStopSec  = 900
Environment     = USE_PRYSM_VERSION=v2.0.2

[Install]
WantedBy    = multi-user.target
```
  

> After you finish editing both of the files, you need to restart systemd:

Reviewer: ```suggestion
After you finish editing both of the files, you need to reload the service unit
```

```
sudo systemctl daemon-reload
```

> Once you do that, the version of prysm beacon and validator are locked in that version, so you need to always update it. If you want to go back to the automatic upgrades after reboot, you just need to remove the `Environment` key.

Reviewer: ```suggestion
Once you do that, the prysm beacon and validator are locked in that version, so you need to always update it. If you want to go back to the automatic upgrades after reboot, you just need to remove the `Environment` key.
```
